src/main/java/edu/rpi/legup/puzzle/nurikabe/rules/SurroundRegionDirectRule.java

```diff
@@ -3,14 +3,23 @@
 import edu.rpi.legup.model.gameboard.Board;
 import edu.rpi.legup.model.gameboard.PuzzleElement;
 import edu.rpi.legup.model.rules.DirectRule;
-import edu.rpi.legup.model.rules.ContradictionRule;
 import edu.rpi.legup.model.tree.TreeNode;
 import edu.rpi.legup.model.tree.TreeTransition;
 import edu.rpi.legup.puzzle.nurikabe.NurikabeBoard;
 import edu.rpi.legup.puzzle.nurikabe.NurikabeCell;
 import edu.rpi.legup.puzzle.nurikabe.NurikabeType;
+import edu.rpi.legup.puzzle.nurikabe.NurikabeUtilities;
+import edu.rpi.legup.utility.DisjointSets;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Set;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import java.awt.*;
 
 public class SurroundRegionDirectRule extends DirectRule {
+    private final static Logger LOGGER = LogManager.getLogger("");

```

Comment:
What is the purpose of `LOGGER`? I don't see it being used anywhere.

Reply:
Sorry that was used in debugging, you can delete that line.